### PR TITLE
feat: restrict confirm/kick of RSVPs

### PR DESCRIPTION
**Pull request by ojeytonwilliams**
- refactor: remove unused command
- test: members cannot kick or confirm
- feat: restrict deleteRsvp and confirmRsvp
- fix: give instance owner rsvp-confirm and delete

- [x] I have read [Chapter's contributing guidelines](https://github.com/freeCodeCamp/chapter/blob/main/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update README.md`).
- [x] My pull request targets the `main` branch of Chapter.

Closes #XXXXX

### CODE REVIEW

File: cypress/support/commands.js [hunk 1]
```diff
@@ -263,6 +238,56 @@ Cypress.Commands.add(
...
+});
+
+Cypress.Commands.add('authedRequest', (options) => {
+  return cy.request({
+    ...options,
+    headers: {
+      Authorization: `Bearer ${window.localStorage.getItem('token')}`,
+    },
+  });
+});
```
Comment (gikf) [inline cypress/support/commands.js:266]: This seems to be duplicate of lines further in the file: https://github.com/freeCodeCamp/chapter/blob/6618f6d10eb0f27ce6933e60a668b3b6b9b63ae9/cypress/support/commands.js#L259-L266

File: cypress/e2e/dashboard/events/[eventId].cy.js [hunk 2]
```diff
@@ -102,5 +102,35 @@ describe('event dashboard', () => {
         cy.get('@rsvps').contains(userName);
       });
     });
+
+    it('prevents members from confirming or kicking users', () => {
+      const eventId = 1;
+      // Start as the instance owner to ensure we can find the RSVPs
+      cy.login();
```
Comment (gikf) [inline cypress/e2e/dashboard/events/[eventId].cy.js:109]: Not sure if this is needed explicitly, as `beforeEach` logs in as owner, comment might be enough. :man_shrugging:

**Review by gikf — COMMENTED** [review]
Just a couple of nitpicks. I'm mostly ignoring inconsistencies in roles factories, as those are being refactored anyway.

File: cypress/support/commands.js (hunk 1 above)
Comment (ojeytonwilliams) [reply cypress/support/commands.js:266]: That's what I get for trusting git rebase. Thanks!

**Review by gikf — APPROVED** [review]
Looks good :+1:

**Pull request merged**